### PR TITLE
Cleanup/flag case consolidation and cleanup

### PR DESCRIPTION
**Pull request by shairozan**

### CODE REVIEW

**Comment by shairozan** [comment]
@dpastoor 

This is purely a cleanup of documentation branch. No code changes. I went through and removed the global flags from each subsequent child command since it keeps the flags from just being repeated page after page.

**Comment by shairozan** [comment]
Added a fix for #150 into this branch.

File: docs/bbi/nonmem/run/local/local.md [hunk 1]
```diff
@@ -9,30 +9,16 @@ Specifies to run the targeted model on the local machine
 ### Options
 
 ```
-      --cacheDir string       directory path for cache of nonmem executables for NM7.4+
-      --cacheExe string       name of executable stored in cache
```
Comment (Dreznel) [inline docs/bbi/nonmem/run/local/local.md]: Do all of these options no longer exist?

Comment (shairozan) [reply docs/bbi/nonmem/run/local/local.md]: They're not implemented anymore. They were from the original incarnation of Babylon, but are not actually implemented anywhere now.

**Pull request merged**